Storage/SQL/BrowserSchema.swift

```diff
@@ -1014,13 +1015,14 @@ open class BrowserSchema: Schema {
             }
         }
 
-        if from < 20 && to >= 20 {
-            if !self.run(db, queries: [
-                "DROP VIEW IF EXISTS \(ViewBookmarksBufferOnMirror)",
-                self.bufferBookmarksView]) {
-                return false
-            }
-        }
+// That view is re-created later
```

Comment:
Indenting on these first lines.

I was going to suggest writing a block comment instead, but I see commented-out migrations below, so fine.